modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/notifications/NotificationsInfo.js

```diff
@@ -119,19 +119,27 @@ const NotificationsInfo = ({
 	const [notificationName, setNotificationName] = useState(
```

Comment:
It is crashing on this line when clicking on the "New Notification" button at the bottom. Although you did not change this line directly, it is working on the master branch. Some other change affected this. `notificationIndex` is iterated in Notifications.js and represents the index of the section when you create a new notification by clicking on the "New Notification" button at the bottom.